Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -271,15 +271,7 @@ public protocol SortedEquatable {
 
 extension BSONDocument: SortedEquatable {
     public func sortedEquals(_ other: BSONDocument) -> Bool {
-        guard self.buffer.readableBytes == other.buffer.readableBytes else {
-            return false
-        }
-        for (k, v) in self {
-            guard let otherValue = other[k], v.sortedEquals(otherValue) else {
-                return false
-            }
-        }
-        return true
+        self.equalsIgnoreKeyOrder(other)
     }
 }
 
```

Comment:
the `BSON` portion (just below this line) still needs to be duplicated since we don't export that from the BSON library. I think that's fine since it's simple enough, and I don't think the method makes all that much sense on `BSON` outside of the contexts we use it in to include in the public API.